<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin_index_conf</class>
 <widget class="QDialog" name="admin_index_conf">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>566</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>361</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string>权重赋值方法</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>361</width>
      <height>451</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::South</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>○ 手动赋值法</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget_modify">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>341</width>
        <height>391</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>0</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton
{	
	color: rgb(0, 170, 127);
}</string>
      </property>
      <property name="text">
       <string>应用</string>
      </property>
     </widget>
     <widget class="QLabel" name="success_tip">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel
{
	color:rgb(255, 0, 0);
}</string>
      </property>
      <property name="text">
       <string>      权值更改成功！</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>△ 层次分析赋值法</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>♢ 环比分析法</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="index_write_access">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>510</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{	
	
	color: rgb(255, 0, 0);
}</string>
   </property>
   <property name="text">
    <string>更新到数据库</string>
   </property>
   <property name="icon">
    <iconset resource="../file.qrc">
     <normaloff>:/png/rc/button/1 (7).png</normaloff>:/png/rc/button/1 (7).png</iconset>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>520</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color: rgb(83, 83, 83);
}</string>
   </property>
   <property name="text">
    <string>当前节点：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="node_show">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>520</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_access">
   <property name="geometry">
    <rect>
     <x>273</x>
     <y>500</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{	
	color: rgb(0, 170, 127);
}</string>
   </property>
   <property name="text">
    <string>重建当前指标体系</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="deletindex">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{	
	color: rgb(0, 170, 127);
}</string>
   </property>
   <property name="text">
    <string>删除指标</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>391</width>
     <height>441</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QComboBox" name="index_select">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>134</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox
{	
	
	color: rgb(255, 170, 15);
}</string>
   </property>
   <item>
    <property name="text">
     <string>服务器设备指标体系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>控制设备指标体系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>显控设备指标体系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>网络架构指标体系</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>信息流指标体系</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>23</y>
     <width>148</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel
{
	background-color: rgb(85, 85, 85);
}</string>
   </property>
   <property name="text">
    <string>当前选择的指标体系 ：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="inputIndex">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="add_child_index">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{	
	color: rgb(0, 170, 127);
}</string>
   </property>
   <property name="text">
    <string>添加叶指标</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="modifyIndex">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>530</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{	
	color: rgb(0, 170, 127);
}</string>
   </property>
   <property name="text">
    <string>修改指标名</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>260</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton
{
	background-color:transparent
}</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="icon">
    <iconset resource="../file.qrc">
     <normaloff>:/png/rc/link.png</normaloff>:/png/rc/link.png</iconset>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="svg_view">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>220</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QRadioButton
{
	background-color: rgb(85, 85, 85);
}</string>
   </property>
   <property name="text">
    <string>切换视图</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../file.qrc"/>
 </resources>
 <connections/>
</ui>
